spec/controllers/admin/rooms_controller_spec.rb

```diff
@@ -0,0 +1,188 @@
+require 'spec_helper'
+
+describe Admin::RoomsController do
+  let!(:admin) { create(:admin) }
+  let(:conference) { create(:conference) }
+  let!(:venue) { create(:venue, conference: conference) }
+  let(:room) { create(:room, venue: venue, size: 4) }
+
+  context 'non admin user is signed in' do
```

Comment:
I added it here after [this diff](https://github.com/openSUSE/osem/pull/837#diff-for-comment-55135403).